puppet_forge.gemspec

```
@@ -23,7 +23,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "faraday", "~> 2.0"
   spec.add_runtime_dependency "faraday-follow_redirects", "~> 0.3.0"
   spec.add_dependency "semantic_puppet", "~> 1.0"
-  spec.add_dependency "minitar", "< 1.0.0 "
+  spec.add_dependency "minitar", "~> 1.0.0"
```

There are already two bugfix releases for minitar, so personally I would do:
```suggestion
  spec.add_dependency "minitar",  '~> 1.0', '>= 1.0.2'
```